Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkDemos/SR-ResearchDemos/PursuitTarget/EyeLink_PursuitTarget.m

```diff
@@ -0,0 +1,388 @@
+        Eyelink('Command', 'record_status_message "TRIAL %d/%d %s"', i, length(trials), char(type(i)));
+        
+        % Draw target trajectory lines on the EyeLink Host PC display.
+        % See section 25.7 'Drawing Commands' in the EyeLink Programmers Guide manual
+        Eyelink('SetOfflineMode');% Put tracker in idle/offline mode before drawing Host PC graphics and before recording
+        Eyelink('Command', 'clear_screen 0'); % Clear Host PC display from any previus drawing
+        Eyelink('Command', 'draw_line %d %d %d %d 15', (width/2)-amplitudeX, height/2, (width/2)+amplitudeX, height/2);
+        Eyelink('Command', 'draw_line %d %d %d %d 15', (width/2), height/2-amplitudeY, (width/2), height/2+amplitudeY);
+        WaitSecs(0.1);  % Allow some time for drawing
+        
+        % Perform a drift check/correction.
+        % Present the drift-check/correction target at each trial's start x y pursuit target location
+        EyelinkDoDriftCorrection(el, round(x), round(y));
+        
+        %STEP 5.3: START RECORDING
+        
+        % Put tracker in idle/offline mode before recording. Eyelink('SetOfflineMode') is recommended 
+        % however if Eyelink('Command', 'set_idle_mode') is used allow 50ms before recording as shown in the commented code:        
+        % Eyelink('Command', 'set_idle_mode');% Put tracker in idle/offline mode before recording
+        % WaitSecs(0.05); % Allow some time for transition            
+        Eyelink('SetOfflineMode');% Put tracker in idle/offline mode before recording
+        Eyelink('StartRecording'); % Start tracker recording
+        WaitSecs(0.1); % Allow some time to record a few samples before presenting first stimulus
+        
+        % STEP 5.4: PRESENT TARGET; CREATE PURSUIT TARGET AND DYNAMIC IA FOR DATAVIEWER; STOP RECORDING 
+        
+        while 1 % loop until error, or timeout
+            % Check that eye tracker is  still recording. Otherwise close and transfer copy of EDF file to Display PC
+            err = Eyelink('CheckRecording');
+            if(err ~= 0)
+                fprintf('EyeLink Recording stopped!\n');
+                % Transfer a copy of the EDF file to Display PC
+                Eyelink('SetOfflineMode');% Put tracker in idle/offline mode
+                Eyelink('CloseFile'); % Close EDF file on Host PC
+                Eyelink('Command', 'clear_screen 0'); % Clear trial image on Host PC at the end of the experiment
+                WaitSecs(0.1); % Allow some time for screen drawing
+                % Transfer a copy of the EDF file to Display PC
+                transferFile; % See transferFile function below
+                cleanup; % Abort experiment (see cleanup function below)
+                return
+            end
+            % Prepare and show the screen.
+            % Enable alpha blending for drawing of smooth points
+            Screen('BlendFunction', window, GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);

```

Comment:
Redundant call to Blendfunction? See above.

Reply:
resolved in 0abf0cbe36691edea3a6d49a87b79149ad7e6f94